reference/reference-environment-variables.md

```diff
@@ -59,6 +59,7 @@ So you can alter the build&start process for your application.
 |[CC_METRICS_PROMETHEUS_PASSWORD]({{< ref "administrate/metrics/overview.md#publish-your-own-metrics" >}}) | Define the password for the basic auth of the Prometheus endpoint | |
 |[CC_METRICS_PROMETHEUS_RESPONSE_TIMEOUT]({{< ref "administrate/metrics/overview.md#publish-your-own-metrics" >}}) | Define the timeout in seconds to collect the application metrics. This value **must** be below 60 seconds as data are collected every minutes | `3` |
 |[CC_CLAMAV]({{< ref "administrate/clamav.md" >}}) | Start the clamav and clamav-freshclam services (the database is updated every 2 hours). WARNING: Clamscan consumes a lot of resources (~ 1GB of memory), make sure you have a scaler with enough memory to avoid OOM. | `false` |
+|CC_NODE_VERSION| Allows to set Node.js version on others application types (Don't use it for Node.js applications) | |
```

Comment:
I'm not really sure about the wording but I can't really think of something better either.

Reply:
How could I rewrite it ?